src/test/resources/slack-messages/expected-github-slack-request.json

```diff
@@ -1,3 +1,3 @@
 {
-  "text": "Hello team!\nHere are the pull requests to review today:\n    *tdr-dev-documentation*\n         TomJKing: <https://github.com/nationalarchives/tdr-dev-documentation/pull/1|Add draft Terraform documentation>\n    *tdr-prototype-mvc*\n         ian-hoyle: <https://github.com/nationalarchives/tdr-prototype-mvc/pull/33|Add filters>\n         suzannehamilton: <https://github.com/nationalarchives/tdr-prototype-mvc/pull/25|Add file upload page>\n"
+  "text" : "\nHello team!\nThese pull requests have had no activity for more than two days:\n    *tdr-dev-documentation*\n         TomJKing: <https://github.com/nationalarchives/tdr-dev-documentation/pull/1|Add draft Terraform documentation> *Updated 745 days ago*\n    *tdr-prototype-mvc*\n         ian-hoyle: <https://github.com/nationalarchives/tdr-prototype-mvc/pull/33|Add filters> *Updated 738 days ago*\n         suzannehamilton: <https://github.com/nationalarchives/tdr-prototype-mvc/pull/25|Add file upload page> *Updated 761 days ago*\n    *prototype-server*\n         ian-hoyle: <https://github.com/nationalarchives/prototype-server/pull/33|Add filters> *Updated 738 days ago*\n         suzannehamilton: <https://github.com/nationalarchives/prototype-server/pull/25|Add file upload page> *Updated 761 days ago*\n\n"
```

Comment:
Is the `\n` at the start correct?

Reply:
Yep. It didn't before because the test PRs which had no activity for more than 2 days were marked as private so weren't showing. Now they are so [this line](https://github.com/nationalarchives/pull-request-monitor/blob/master/src/main/scala/uk/gov/nationalarchives/digitalarchiving/mergerequestmonitor/notifications/SlackMessageFormatter.scala#L28) happens. It's what we're doing currently so it's ok.